src/app/search-navbar/search-navbar.component.html

```diff
@@ -4,9 +4,9 @@
       <input #searchInput [@toggleAnimation]="isExpanded" [attr.aria-label]="('nav.search' | translate)" name="query"
              formControlName="query" type="text" placeholder="{{searchExpanded ? ('nav.search' | translate) : ''}}"
              class="d-inline-block bg-transparent position-absolute form-control dropdown-menu-right p-1" [attr.data-test]="'header-search-box' | dsBrowserOnly">
-      <a class="submit-icon" [routerLink]="" (click)="searchExpanded ? onSubmit(searchForm.value) : expand()" [attr.data-test]="'header-search-icon' | dsBrowserOnly">
+      <button class="submit-icon btn btn-link btn-link-inline" type="button" (click)="searchExpanded ? onSubmit(searchForm.value) : expand()" [attr.data-test]="'header-search-icon' | dsBrowserOnly">
```

Comment:
I'm getting an accessibility error here now (from the axe DevTools scanner plugin in Chrome) which says "Buttons must have discernible text".  The issue appears to be that this has no text within it, or `aria-label` or `title`.  

Could we add a `title` or `aria-label` attribute here?  That should fix the accessibility issue.

Reply:
Added aria-label